standalone/ansible/roles/logging/templates/etc/logrotate.d/rails.j2

```diff
@@ -0,0 +1,17 @@
+/home/deploy/apps/*/current/log/*.log {
+  su deploy deploy
+  daily
+  rotate 14
+  missingok
+  notifempty
+  compress
+  delaycompress
+  copytruncate
+  dateext
+  lastaction
+    HOSTNAME=`hostname`
+{% for server in groups.storage %}
+    rsync -a /home/deploy/apps/{{ app_name }}/current/log/*.gz {{ deploy_user }}@{{ server }}:~/logs/rails/
```

Comment:
Neat, so `rsync` is installed on the web server machines, and the web servers are configured to push their logs to the `storage` server(s)? Neat. I'm assuming that the `deploy` user's SSH keys are present and authorized across all these machines, which is why `deploy@server` will work?

Reply:
Yes. This keys are being added here:

https://github.com/simpledotorg/deployment/blob/d7ca44946894d534a011dcaba27714091211e336/standalone/ansible/roles/logging/tasks/main.yml#L12